### ISSUE TITLE
Can't mark more than one sequence for upload

### ISSUE THREAD

**Issue by simonpoole** [mapillary/mapillary_issues#2337]
### Basic information
* Platform:  `Android` 
* Version: `3.01` 
* System/Browser: `Samsung S5 6.01`

### Steps to reproduce behavior

 1. Open upload "tab"
 2. Select sequence
 3. Start upload
 
### Expected behavior

Further sequences can be reviewed (if that worked) and marked for upload.

### Actual behavior

Dialog is shown that only offer options "Cancel" and "Home" selecting the later returns to the upload "tab"....

**Comment by HolgerJeromin** [comment]
Yes, this is annoying. I reported it about ten days ago.

**Comment by HolgerJeromin** [comment]
This really is not a good workflow as often I review a few random sequences on my phone when I cannot upload. Marking them for later was much better.

**Comment by peterneubauer** [comment]
So, the thinking here is that for the time being, it should be superclear for the user what is uploaded, and sequences that are being uploaded should not be changed at the same time, ping @angelday . Right now you either upload all sequences or one by one from the sequence screen, and a proper select-some-sequences will need more design as you probably want to see what sequences are uploading, and cancel/prioritize them.

**Comment by angelday** [comment]
Yes, a more complex and roboust review workflow should be better where you could mark sequences for upload and let them go in one fell swoop.

We have been aware of this and will update once made the required design/development. Thanks for reporting.

**Comment by peterneubauer** [comment]
mapillary_issues is no longer an active repository.
Let’s continue discussing features in our forum! We will import this issue as topic to the Mapillary Forum in the coming weeks. We’ll notify here once it’s been done. See you there!
